lib/proguard-inflater.pro

```diff
@@ -12,5 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# MaterialComponentsViewInflater inflates Material Components rather than their AppCompat counterparts.
--keep class com.google.android.material.theme.MaterialComponentsViewInflater { *; }
+# AppCompatViewInflater reads the viewInflaterClass theme attribute which then reflectively
```

Comment:
Thanks @JakeWharton! It looks like `viewInflaterClass` is technically read by `AppCompatDelegateImpl`:

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-master-dev:appcompat/appcompat/src/main/java/androidx/appcompat/app/AppCompatDelegateImpl.java;l=1420;drc=9502ef7329a3a1b8e9dc4afbd51460ddf545050c

Should the condition be changed to be dependent on `AppCompatDelegateImpl` instead of `AppCompatViewInflater`?

Reply:
My hesitation there is that is a private type whose name is free to change at any time. Whereas `AppCompatViewInflater` is the public type that `MaterialComponentsViewInflater` extends and can never change.

Reply:
Sounds good, will submit the PR internally and sync the change back out here.